README.rst

```diff
@@ -84,9 +84,9 @@ as the default does.
 Making your own XBlock
 ----------------------
 
-Making an XBlock can be as simple as creating a Python class with a few
-specific methods.  The ``thumbs`` XBlock demonstrates an XBlock with state,
-views, and input handling.
+Making an XBlock involves creating a Python class that conforms to the XBlock 
+specification. See the sample_xblocks directory for examples and the 
+XBlock tutorial for a full walk-through: http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/index.html
```

Comment:
Please use ReST syntax for making links with English link text, and a URL behind the scenes:
```
See the sample_xblocks directory for examples and `the XBlock tutorial`_ for a full walk-through.

.. _the XBlock tutorial: http://edx.readthedocs.org/projects/xblock-tutorial
```